e2e/test/helpers/ConsoleEventListener.cs

```diff
@@ -10,16 +9,20 @@ namespace System.Diagnostics.Tracing
     public sealed class ConsoleEventListener : EventListener
     {
         // Configure this value to filter all the necessary events when OnEventSourceCreated is called.
-        // OnEventSourceCreated is triggered as soon as the EventListener is registered and an event source is created.
-        // So trying to configure this value in the ConsoleEventListener constructor does not work.
+        // The EventListener base class constructor creates an event listener in which all events start off turned off.
```

Comment:
"start off turned off" sounds wrong.

Reply:
I thought so too, but it is the exact wording from the doc: https://github.com/dotnet/runtime/blob/6696065ab0f517f5a9e5f55c559df0010a816dbe/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs#L4009-L4018.
How about: "The EventListener base class constructor creates an event listener in which all events ~start off turned off~ are disabled by default."